### DISCUSSION TITLE
interrupt_stop_event

### DISCUSSION THREAD

**Discussion by tgdonovan** [KoljaB/RealtimeSTT#61] [Q&A] [answered]
I just started playing with this library. Thank you, it shows a lot of work and thought was put into it.

Probably me not understanding, but if I use     recorder = AudioToTextRecorder(**recorder_config) and then a while true loop, is there a way to interrupt the recording. I see the interrup_stop_event but don't see a way to interact with that from outside of the class. 

What am I missing. And again, thank you for this.

**Comment by KoljaB** [comment]
There is the abort method that should handle this. It's quite fresh implemented though and not well tested.

**Reply by tgdonovan** [reply, marked as answer]
Thank you. Appreciate it.
